apps/spruce/src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -133,8 +137,9 @@ export const getFormSchema = (
             gitTagVersionsTitle: {
               type: "null",
               title: "Trigger Versions With Git Tags",
-              description:
-                "If an authorized user pushes a tag that matches a specific regex, then a version will be created from this alias. Note that project admins are not authorized by default; they must explicitly be given this permission.",
+              description: `If an authorized user pushes a tag that matches a specific regex, then a version will be created from this alias. 
```

Comment:
Confirmed that this worked 
<img width="698" alt="image" src="https://github.com/user-attachments/assets/bd66dfbb-e2da-4585-bcfa-3b0b3cdec6ec" />